apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/ReadTests.cs

```diff
@@ -104,8 +104,8 @@ public class ReadTests : IClassFixture<TestDatabaseFixture>
         [Fact]
         public async Task BadDbName()
         {
-            string connectionString = "Data Source=" + _testFixture.TestProjectName + "/"
-                + _testFixture.TestInstanceName + "/" + "badjuju";
+            string connectionString = "Data Source=projects/" + _testFixture.TestProjectName + "/instances/"
```

Comment:
Possibly use an interpolated string literal? I think that would be easier to understand. You could potentially suck in local variables first:

```csharp
string project = _testFixture.TestProjectName;
string instance = _testFixture.TestInstanceName;

string connectionString = $"Data Source=projects/{project}/instances/{instance}/databases/badjuju";
```

(Ditto for other changes using concatenation.)